core/variant/callable.cpp

```diff
@@ -124,9 +124,22 @@ Callable Callable::unbind(int p_argcount) const {
 bool Callable::is_valid() const {
 	if (is_custom()) {
 		return get_custom()->is_valid();
-	} else {
-		return get_object() && get_object()->has_method(get_method());
 	}
+
+	Object *target = get_object();
+	if (target) {
+		if (target->has_method(get_method())) {
+			return true;
+		}
+		// Script.has_method() and Object::has_method() have different semantics.
+		// In order to check if a script (i.e., a class) has some method it has
+		// to be checked
```

Comment:
```suggestion
		// In order to check if a script (i.e., a class) has some method it has
		// to be checked explicitly.
```

Reply:
Applied, plus changed a bit the wording to make it more visually pleasant.

PS. I swear I abide by the comments-must-end-with-a-period rule (I'm even starting to get used to it), but this time the period went to the limbo.